File: app/src/main/java/com/example/laksh_pc/voteapp/list.java
package com.example.laksh_pc.voteapp;

import android.content.Intent;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.view.ViewGroup;
import android.widget.AdapterView;
import android.widget.BaseAdapter;
import android.widget.ImageView;
import android.widget.ListView;
import android.widget.TextView;

public class list extends AppCompatActivity {

    ListView mlistview;
    int[] images = {R.drawable.pres,R.drawable.gov};
    String[] names = {"Chairman","vise chairman"};
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_list);
        mlistview = (ListView) findViewById(R.id.listview);
        CustomAdapter customAdapter= new CustomAdapter();
        mlistview.setAdapter(customAdapter);
        mlistview.setOnItemClickListener(new AdapterView.OnItemClickListener() {
            @Override
            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
                switch (position){
                    case 0:
                        Intent l  = new Intent(getApplicationContext(),pres.class);
                        startActivity(l);
                        finish();
                        break;
                    case 1:
                        Intent b = new Intent(getApplicationContext(),gov.class);
                        startActivity(b);
                        finish();
                        break;
                }
            }
        });


    }
    class CustomAdapter extends BaseAdapter{

        @Override
        public int getCount() {
            return images.length;
        }

        @Override
        public Object getItem(int position) {
            return null;
        }

        @Override
        public long getItemId(int position) {
            return 0;
        }

        @Override
        public View getView(int position, View convertView, ViewGroup parent) {
            View view = getLayoutInflater().inflate(R.layout.itemrow, null);
            ImageView mimageview = (ImageView) view.findViewById(R.id.imageView8);
            TextView mtextview = (TextView) view.findViewById(R.id.textView9);

            mimageview.setImageResource(images[position]);
            mtextview.setText(names[position]);
            return view;
        }
    }

}

File: app/src/main/java/com/example/laksh_pc/voteapp/List.kt
package com.example.laksh_pc.voteapp

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.view.ViewGroup
import android.widget.AdapterView.OnItemClickListener
import android.widget.BaseAdapter
import android.widget.ImageView
import android.widget.ListView
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class List : AppCompatActivity() {
    var mlistview: ListView? = null
    var images = intArrayOf(R.drawable.pres, R.drawable.gov)
    var names = arrayOf("Chairman", "vise chairman")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_list)
        mlistview = findViewById<View>(R.id.listview) as ListView
        val customAdapter = CustomAdapter()
        mlistview!!.adapter = customAdapter
        mlistview!!.onItemClickListener = OnItemClickListener { parent, view, position, id ->
            when (position) {
                0 -> {
                    val l = Intent(applicationContext, VoteListFragment::class.java)
                    startActivity(l)
                    finish()
                }
                1 -> {
                    val b = Intent(applicationContext, VoteListVCFragment::class.java)
                    startActivity(b)
                    finish()
                }
            }
        }
    }

    internal inner class CustomAdapter : BaseAdapter() {
        override fun getCount(): Int {
            return images.size
        }

        override fun getItem(position: Int): Any? {
            return null
        }

        override fun getItemId(position: Int): Long {
            return 0
        }

        override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
            val view = layoutInflater.inflate(R.layout.itemrow, null)
            val mimageview = view.findViewById<View>(R.id.imageView8) as ImageView
            val mtextview = view.findViewById<View>(R.id.textView9) as TextView
            mimageview.setImageResource(images[position])
            mtextview.text = names[position]
            return view
        }
    }
}